app/web_hub.go

```diff
@@ -415,6 +411,16 @@ func (h *Hub) Start() {
 			case webCon := <-h.register:
 				connections.Add(webCon)
 				atomic.StoreInt64(&h.connectionCount, int64(len(connections.All())))
+				// Send hello message.
+				if webCon.IsAuthenticated() {
+					msg := model.NewWebSocketEvent(model.WEBSOCKET_EVENT_HELLO, "", "", webCon.UserId, nil)
+					msg.Add("server_version", fmt.Sprintf("%v.%v.%v.%v",
+						model.CurrentVersion,
+						model.BuildNumber,
+						webCon.App.ClientConfigHash(),
+						webCon.App.License() != nil))
+					webCon.Send <- msg
```

Comment:
I think the `SendHello` function can remain, just need to unexport it and pass it the `WebConn`. 
I understand it's called only here but since we are inside a pretty nested case it would look a bit more readable.

Reply:
Yeah .. my only concern is if somebody calls it from outside this goroutine. But that seems very unlikely.

Reply:
1/5, but I feel like the nesting was better than the `DO NOT call this function ... ` warning :D

Reply:
Should I just remove the comment ? Or revert ? :P

Reply:
I think it's fine -- no need to churn on your part. Just expressing an opinion :)

Reply:
@lieut-data That's a a fair point. 

I thought the semantic of the function plus the fact is was unexported was enough to make it safe.

Maybe we could just move the msg generation in its own function and keep the send in there.
Something like:

```go
webCon.Send <- createHelloMsg()
```

Reply:
Ooh that's a good one. :+1: